app/controllers/invites_controller.rb

```diff
@@ -0,0 +1,55 @@
+class InvitesController < ApplicationController
+  before_action :authenticate_user!
+  before_filter :get_game
+
+  def new
+    @invite = Invite.new
+  end
+
+  def create
+    @game = Game.find(params[:game_id])
+    @invite = @game.invites.create(invite_params.merge(sender: current_user))
+    @invite.sender_id = current_user.id
```

Comment:
yep, done - these two lines were written at two different times. consider it refactored. :)